src/corehost/cli/corehost_context_contract.h

```diff
@@ -66,4 +67,15 @@ static_assert(offsetof(corehost_context_contract, run_app) == 5 * sizeof(size_t)
 static_assert(offsetof(corehost_context_contract, get_runtime_delegate) == 6 * sizeof(size_t), "Struct offset breaks backwards compatibility");
 #pragma pack(pop)
 
+/// Signature of delegate returned by coreclr_delegate_type::component_create_native_delegate
+typedef int (__cdecl *ComponentCreateNativeDelegate)(const pal::char_t * assemblyPathNative /* Fully qualified path to assembly */,
```

Comment:
Have we decided on the encoding of these strings?

Reply:
I chose to be consistent with the rest of the API.  UTF16 on windows, UTF8 on others. Hence `pal::char_t`